embrace-android-otel/src/main/kotlin/io/embrace/android/embracesdk/internal/otel/sdk/id/OtelIds.kt

```diff
@@ -1,24 +1,25 @@
+@file:OptIn(ExperimentalApi::class)
+
 package io.embrace.android.embracesdk.internal.otel.sdk.id
 
-import io.embrace.opentelemetry.kotlin.aliases.OtelJavaIdGenerator
-import io.embrace.opentelemetry.kotlin.aliases.OtelJavaSpanId
+import io.embrace.opentelemetry.kotlin.ExperimentalApi
+import io.embrace.opentelemetry.kotlin.factory.createCompatSdkFactory
 
 object OtelIds {
-
-    private val generator = OtelJavaIdGenerator.random()
+    private val tracingIdFactory = createCompatSdkFactory().tracingIdFactory
```

Comment:
`createCompatSdkFactory` shouldn't really be public, I'll address this in a separate PR in the opentelemetry-kotlin repo. Basically we'll want to get an `OpenTelemetry` instance as that implements `SdkFactory`.